README.md

```diff
@@ -51,3 +51,22 @@ npm link
 ```
 
 Doing the above (and repeating the last step for any change to the CLI's source) will introduce a `zimlet` command globally.
+
+
+## Custom Configuration
+
+#### Webpack
+
+To customize webpack create a `zimlet.config.js` file in the root of your Zimlet which exports a function that will change webpack's config.
```

Comment:
- [ ] You should specify that you can also override it from `env.config` which appears supported in the code below

Reply:
Noted this in the `README` and as a CLI option.